bagger/README.md

```diff
@@ -0,0 +1,229 @@
+B = Bagger(workflow, output_dir, delete, dart_command, config, log)
+
+# Run DART on the package specified with path
+run = B.run_dart(path)
+```
+
+## Configuration
+
+ReBACH-Bagger uses [TOML-based](https://toml.io/en/) configuration files found
+in the config directory. The `default.example.toml` file in the config
+directory contains the configuration variables expected by ReBACH-Bagger. By
+default, the program will look for a config file named `default.toml` in the
+`bagger/config` directory. A config file can also be specified at
+runtime using `--config` or `-c`. Paths in the config file are all relative to
+the directory from which the script is called. Using absolute paths may be
+advisable if ReBACH-Bagger will be imported as a module.
+
+### Defaults
+
+The `Defaults` section of the configuration file contains keys that define the
+program's execution environment. The keys defined here can also be set at
+runtime using the command line options described above. Command line
+configuration options are first in precendce, meaning values passed with the
+command line will override values set in the config file.
+
+```toml
+[Defaults]
+output_dir = "out" # DART will output generated bags here; must be writable
+workflow = "default_workflow.json" # Path to the DART workflow file
+dart_command = "dart-runner" # Command or path to DART executable
+```
+
+### Logging
+
+ReBACH-Bagger logs errors, debug messages, and DART output to disk.
+
+```toml
+[Logging]
+log_dir = "logs" # Logger will write logs to this directory
+logfile_prefix = "ReBACH-Bagger" # Log filename prefix
+```
+
+### Wasabi
+
+Both DART and ReBACH-Bagger use the `access_key` and `secret_key` credentials defined in the
+configuration to authenticate to Wasabi (see below for details on how these variables are used in
+DART). However, only ReBACH-Bagger uses the other variables defined here to access the endpoint for
+the purpose of checking for duplicate bags. DART uses the storage configuration embedded in the
+workflow JSON file for selecting the correct endpoint. Verify that these values match.
+
+```toml
+[Wasabi]
+name = "***override***"
+host = "***override***"
+bucket = "***override***"
+host_bucket = "***override***"
+access_key = "***override***"
+secret_key = "***override***"
+```
+
+## Metadata Configuration
+
+ReBACH-Bagger can apply a configurable set of metadata tags to the bags it generates. These tags
+are defined in the `Metadata` section of the configuration file. The tags are defined with the
+following schema:
+
+`# tag-file.Tag-Name = "Tag value"`
+
+The `tag-file` element corresponds to the tag file (e.g. bag-info.txt) in which the tag will be
+placed. Do not include `.txt` as part of the tag-file element.
+
+`Tag-Name` is the name of the metadata tag. Conventionally, tag names are uppercase, dash-separated
+words.
+
+If the `"Tag value"` element is a string, ReBACH-Bagger will simply use the string as the value of
+the tag.
```

Comment:
Revise to explicitly say that Tag value does not necessarily need to be a string. e.g.

> The `"Tag value"` element can be a string or an inline table (see below).  If the `"Tag value"` element is a string, ReBACH-Bagger will simply use the string as the value of the tag.